src/components/AssetIcon.tsx

```diff
@@ -47,7 +50,16 @@ export const AssetIcon = ({
       }}
     />
   ) : isAssetSymbol(symbol) ? (
-    <$AssetIcon src={ASSET_ICON_MAP[symbol]} className={className} alt={symbol} />
+    <div tw="relative flex items-center justify-center">
+      <$AssetIcon src={ASSET_ICON_MAP[symbol]} className={className} alt={symbol} />
+      {chainId && (
+        <img
+          tw="absolute bottom-[-2px] right-[-2px] h-[50%] w-[50%] rounded-9 border border-solid border-color-layer-4"
```

Comment:
I think we can use `em` for bottom/right positioning so it scales with AssetIcon

Reply:
looks like 1px per rem?

Reply:
i think its better to leave the offset in px in this case because the positioning relative to the bounding box (eg u dont want the icon to overflow too much if asseticon gets big)

Reply:
kk. I guess we can handle on a case by case basis. Maybe use a css var with default value of `-2px`? so that it can be altered easily? `--asset-icon-chain-offset` or something?

Reply:
i think its fine to leave it in the tailwind class 😅 simpler!

Reply:
ah yeah I think just adding the offset to the `$AssetIcon` styled.img should do the trick. Then `bottom-[...css-var]` but we can tackle that when necessary

Reply:
otherwise we can't access changing the offset from `<AssetIcon>` usage in other components